Getting Started/Full Node Installation.md

```diff
@@ -45,6 +45,25 @@ of the basic operations work with zero configuration (except the testnet flag).
 $ kaspad --testnet
 ```
 
+### Running a Miner (optional)
+After running kaspad you can launch a miner and start mining blocks.
+
+To run a miner you need to create a keypair to mine into:
+```bash
+$ wallet create --testnet
+```
+The result will look something like this:
+```
+This is your private key, granting access to all wallet funds. Keep it safe. Use it only when sending Kaspa.
+Private key (hex):      547f40e991a115b9cda6a5203a2a6f8031d76c46c6575228d213abf04b57a4e7
```

Comment:
Even though it's testnet, I'd rather not have any real pk/address in the docs.

This will encourage lazy people to use these instead of generating their own, and then won't understand what's going on with their coins, once other lazy people spend them.

I'd zero this and the address out (or do something like 0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef)